subCMLs/README.md

```diff
@@ -0,0 +1,5 @@
+# SubCMLs
+
+Sub CMLs are CatalystML jsons that are intended to be run within another CML json with the runCML operation.  The usefully ness of this is that it allows you to recycle commonly reused combinations of operations.  Effectively making your own transferable operations.
```

Comment:
```suggestion
Sub CMLs are CatalystML jsons that are intended to be run within another CML json with the runCML operation.  The usefulness of this is that it allows you to recycle commonly used transformations.
```

Reply:
Not really sure about the name `subCMLs`...

Reply:
I am open to other suggestions.

Reply:
I'm also not sure we need a top level dir named `subCMLs` what else will be in this dir, besides this readme?